hometask8/controllers/Groups.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+var url = require('url');
+
+
+var Groups = require('./GroupsService');
+
+
+module.exports.groupPost = function groupPost (req, res, next) {
+  var group = req.swagger.params['group'].value;
+  
+
+  var result = Groups.groupPost(group);
+
+  if(typeof result !== 'undefined') {
+    res.setHeader('Content-Type', 'application/json');
+    res.end(JSON.stringify(result || {}, null, 2));
+  }
+  else
+    res.end();
+};
+
+module.exports.groupGetAllGet = function groupGetAllGet (req, res, next) {
+  
+
+  var result = Groups.groupGetAllGet();
+
+  if(typeof result !== 'undefined') {
+    res.setHeader('Content-Type', 'application/json');
+    res.end(JSON.stringify(result || {}, null, 2));
+  }
+  else
+    res.end();
+};
+
+module.exports.groupIdGet = function groupIdGet (req, res, next) {
+  var id = req.swagger.params['id'].value;
+  
+
+  var result = Groups.groupIdGet(id);
+
+  if(typeof result !== 'undefined') {
+    res.setHeader('Content-Type', 'application/json');
+    res.end(JSON.stringify(result || {}, null, 2));
+  }
+  else
+    res.end();
+};
+
+module.exports.groupIdPut = function groupIdPut (req, res, next) {
```

Comment:
As I mentioned in previous PRs, inline exporting is not considered a "best practice" by most of the linting tools.
Consider bringing all the exports down to the bottom of the file.